thali/thalicryptomanager.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var crypto = require('crypto');
+var fs = require('fs');
+var path = require('path');
+
+var pkcs12FileName = '/pkcs12.pfx';
+var password = 'password';
+var certname = 'certname';
+var country = 'country';
+var organization = 'organization';
+var macName = 'SHA256';
+var hashSizeInBytes = 16;
+
+module.exports = {
+
+/**
+* Creates a PKCS12 content, saves it to a file, extracts the public
```

Comment:
This description is not correct. It would be better to use a correct description which is that this function looks for a pkcs12 file in a known location and will create it if it doesn't exist and then extract the public key and return its SHA256 hash.